src/Solvers/solve_for_pressure.jl

```diff
@@ -1,14 +1,14 @@
 using Oceananigans.Operators
 
-function solve_for_pressure!(pressure, solver, arch, grid, U, G, Δt)
+function solve_for_pressure!(pressure, solver, arch, grid, Δt, U★)
     if solver.type isa Channel && arch isa GPU
         ϕ = RHS = solver.storage.storage1
     else
         ϕ = RHS = solver.storage
     end
 
     @launch(device(arch), config=launch_config(grid, :xyz),
-            calculate_pressure_right_hand_side!(RHS, solver.type, arch, grid, U, G, Δt))
+            calculate_pressure_right_hand_side!(RHS, solver.type, arch, grid, Δt, U★))
 
     solve_poisson_equation!(solver, grid)
```

Comment:
Not sure why but one of the Travis builds hits a `ReadOnlyMemoryError` on this line and I get a segmentation fault on my computer at this line.

Perhaps part of the model is corrupted after restoring from checkpoint? I'll try to find out.

Travis build log: https://travis-ci.com/climate-machine/Oceananigans.jl/jobs/294185099#L453